src/sentry/integrations/gitlab/client.py

```diff
@@ -258,6 +258,13 @@ def get_last_commits(self, project_id, end_sha):
         path = GitLabApiClientPath.commits.format(project=project_id)
         return self.get(path, params={"until": end_date})
 
+    def get_commit(self, project_id, sha):
+        """
+        Get the details of a commit
+        See https://docs.gitlab.com/ee/api/commits.html#get-a-single-commit
+        """
+        return self.get(GitLabApiClientPath.commit.format(project=project_id, sha=sha))
```

Comment:
@AniketDas-Tekky can we use get_cached here?

Reply:
mb, I updated the wrong client

Reply:
@AniketDas-Tekky but we can still use `get_cached`

Reply:
updated